sickbeard/providers/tvchaosuk.py

```diff
@@ -102,7 +100,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
             for search_string in search_strings[mode]:
 
                 if mode == 'Season':
-                    search_string = re.sub(ur'(.*)Season', ur'\1Series', search_string)
+                    search_string = re.sub(ur'(.*)S0?', ur'\1Series', search_string)
```

Comment:
@duramato can you add comments with examples of releases that matches it ?

what abou releases like Show1.Season.6...... ?